graphene/types/decimal.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from decimal import Decimal as _Decimal
+
+from graphql.language import ast
+
+from .scalars import Scalar
+
+
+class Decimal(Scalar):
+    """
+    The `Decimal` scalar type represents a python Decimal.
+    """
+
+    @staticmethod
+    def serialize(dec):
+        if isinstance(dec, str):
+            dec = _Decimal(dec)
+        assert isinstance(dec, _Decimal), 'Received not compatible Decimal "{}"'.format(
+            repr(dec)
+        )
+        return str(dec)
+
+    @classmethod
+    def parse_literal(cls, node):
+        if isinstance(node, ast.StringValue):
+            return cls.parse_value(node.value)
+
+    @staticmethod
+    def parse_value(value):
+        try:
+            return _Decimal(value)
+        except ValueError:
```

Comment:
Shouldn't it check for `decimal.DecimalException` here, as for example [Django does](https://github.com/django/django/blob/master/django/forms/fields.py#L346)? I've copied your changes to my repo (as it hasn't been released yet) and I'm trying to use it in my mutations, but when providing invalid values like an empty string I'm getting:
```
{
  "errors": [
    {
      "message": "[<class 'decimal.ConversionSyntax'>]"
    }
  ]
}
```

Reply:
@maarcingebala can you submit a pull request with this change so it can be reviewed?